chronograf.go

```diff
@@ -111,9 +112,10 @@ type TimeSeries interface {
 
 // Role is a restricted set of permissions assigned to a set of users.
 type Role struct {
-	Name        string      `json:"name"`
-	Permissions Permissions `json:"permissions,omitempty"`
-	Users       []User      `json:"users,omitempty"`
+	Name         string      `json:"name"`
+	Permissions  Permissions `json:"permissions,omitempty"`
+	Users        []User      `json:"users,omitempty"`
+	Organization string      `json:"organization,string,omitempty"`
```

Comment:
Should all of these orgs be []string?

Reply:
I think that is correct, but I'd like to tackle it as a separate PR. Created https://github.com/influxdata/chronograf/issues/2200